go-controller/pkg/ovn/controller/admin_network_policy/admin_network_policy.go

```diff
@@ -263,12 +263,11 @@ func (c *Controller) convertANPPeersToIPs(anp *adminNetworkPolicyState) error {
 // This function also takes care of populating the adminNetworkPolicyPeer.namespaces cache
 // It also adds up all the peerIPs that are supposed to be present in the created AS and returns them on
 // a per-rule basis so that the actual ops to transact these into the AS can be constructed using that
-func (c *Controller) convertANPPeersToIPSet(peers []*adminNetworkPolicyPeer) (sets.Set[string], error) {
-	peerIPs := sets.Set[string]{}
+func (c *Controller) convertANPPeersToIPSet(peers []*adminNetworkPolicyPeer, peerIPs sets.Set[string]) error {
```

Comment:
At this point, would it make it easier to understand if we reframe this method as 
```
func (c *Controller) expandANPRulePeersToIPSet(rule *gressRule) error
```

Reply:
+1 I have been struggling to name these functions correctly. that sgtm

Reply:
done. do you think I should change `convertANPPeersToIPs` on the higher level or should I leave that as is.